### DISCUSSION TITLE
Data Error!

### DISCUSSION THREAD

**Discussion by clundin** [HASwitchPlate/HASPone#169] [Q&A] [answered]
I did the HASPone pdate to 1.05. Then firmware update and I should probably have stayed away from update LCD. It was 3 and now I get a Data Error! and LCD version is 0... 

This was the URL
https://raw.githubusercontent.com/HASwitchPlate/HASPone/main/Nextion_HMI/HASwitchPlate.tft

**Comment by aderusha** [comment, marked as answer]
Can you try the LCD flash process again?  Just use the default URL and run the LCD update and give it another go.

**Reply by clundin** [reply]
I downloaded the file and tried the LCD flash process from file this time. It worked! Back to LCD firmware 3 - so I guess I already had the latest LCD firmware version...

**Reply by aderusha** [reply]
Great news!  In general, if an update is available you’ll see a big green notice in the web UI.  It checks for updates on boot and then every 12 hours after that.
